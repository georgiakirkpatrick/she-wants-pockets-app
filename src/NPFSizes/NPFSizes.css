.NPFSizes .FormCheckboxSection__options-container {
    height: 350px;
}