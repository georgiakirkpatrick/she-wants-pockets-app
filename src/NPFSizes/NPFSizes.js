import React from 'react'
import FormPage from '../FormPage/FormPage'
import FormCheckboxSection from '../FormCheckboxSection/FormCheckboxSection'
import FormPromptWithSub from '../FormPromptWithSub/FormPromptWithSub'
import NPFFooter from '../NPFFooter/NPFFooter'
import sizeData from '../SIZES'

const NPFSizes = props => {
    const sizeChange = event => {
        props.setSelectedSizeOptions({...props.selectedSizeOptions, [event.target.id]: !props.selectedSizeOptions[event.target.id]})
    }

    return (
        <div>
            <FormPage title='Sizes'>
                <FormPromptWithSub 
                    prompt='Select all size options available on this product page'kk
                    promptSubtitle='Include sold out sizes'
                />

                <FormCheckboxSection 
                    prompt='Standard US sizing'
                    options={sizeData.women.dressSizes.standard.map(size => (
                        {
                            id: size.US.id,
                            text: `${size.US.text}, ${size.alpha}`
                        }
                    ))}
                    selectedOptions={props.selectedSizeOptions}
                    handleChange={event => sizeChange(event)}
                />

                <FormCheckboxSection 
                    prompt={'Petite US sizing'} 
                    options={sizeData.women.dressSizes.petite.map(size => (
                        {
                            id: size.US.id,
                            text: `${size.US.text}, ${size.alpha}`
                        }
                    ))}
                    selectedOptions={props.selectedSizeOptions}
                    handleChange={event => sizeChange(event)}
                />

                <FormCheckboxSection 
                    prompt={'Tall US sizing'} 
                    options={sizeData.women.dressSizes.tall.map(size => (
                        {
                            id: size.US.id,
                            text: `${size.US.text}, ${size.alpha}`
                        }
                    ))}
                    selectedOptions={props.selectedSizeOptions}
                    handleChange={event => sizeChange(event)}
                />

                <FormCheckboxSection 
                    prompt={'Maternity US sizing'} 
                    options={sizeData.women.dressSizes.maternity.map(size => (
                        {
                            id: size.US.id,
                            text: `${size.US.text}, ${size.alpha}`
                        }
                    ))}
                    selectedOptions={props.selectedSizeOptions}
                    handleChange={event => sizeChange(event)}
                />
            </FormPage>

            <NPFFooter 
                buttons='prevNext' 
                previousButton={() => props.setPage(props.currentPage - 1)} 
                nextButton={() => props.setPage(props.currentPage + 1)} 
            />
        </div>
    )
}

NPFSizes.defaultProps = {
    currentPage: 0,
    selectedSizeOptions: [],
    setPage: () => {},
    sizeChange: () => {},
    sizes: []
}

export default NPFSizes