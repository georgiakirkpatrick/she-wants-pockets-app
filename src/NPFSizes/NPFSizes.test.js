import React from 'react'
import ReactDOM from 'react-dom'
import NPFSizes from './NPFSizes'
import renderer from 'react-test-renderer'

describe ('NPFSizes', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<NPFSizes />, div
        )
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<NPFSizes />)
            .toJSON()
        expect(tree).toMatchSnapshot()    
    })
})