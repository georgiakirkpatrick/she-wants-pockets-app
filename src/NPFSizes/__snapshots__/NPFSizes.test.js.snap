// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NPFSizes renders the UI as expected 1`] = `
<div>
  <section
    className="FormPage"
  >
    <h1>
      Sizes
    </h1>
    <form>
      <div
        className="FormPromptWithSub"
      >
        <p
          className="FormPromptWithSub__prompt"
        >
          Select all size options available on this product page
        </p>
        <p
          className="FormPromptWithSub__subtitle"
        >
          Include sold out sizes
        </p>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Standard US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-00-XXS"
              name="dress-us-standard-00-XXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-00-XXS"
            >
              00, XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-0-xs"
              name="dress-us-standard-0-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-0-xs"
            >
              0, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-2-xs"
              name="dress-us-standard-2-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-2-xs"
            >
              2, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-4-s"
              name="dress-us-standard-4-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-4-s"
            >
              4, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-6-s"
              name="dress-us-standard-6-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-6-s"
            >
              6, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-8-m"
              name="dress-us-standard-8-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-8-m"
            >
              8, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-10-m"
              name="dress-us-standard-10-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-10-m"
            >
              10, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-12-l"
              name="dress-us-standard-12-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-12-l"
            >
              12, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-14-l"
              name="dress-us-standard-14-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-14-l"
            >
              14, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-16-XL"
              name="dress-us-standard-16-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-16-XL"
            >
              16, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-18-XL"
              name="dress-us-standard-18-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-18-XL"
            >
              18, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-20-2X"
              name="dress-us-standard-20-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-20-2X"
            >
              20, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-22-2X"
              name="dress-us-standard-22-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-22-2X"
            >
              22, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-24-3X"
              name="dress-us-standard-24-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-24-3X"
            >
              24, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-26-3X"
              name="dress-us-standard-26-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-26-3X"
            >
              26, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-standard-28-4X"
              name="dress-us-standard-28-4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-standard-28-4X"
            >
              28, 4X
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Petite US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-00-XXS"
              name="dress-us-petite-00-XXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-00-XXS"
            >
              00, XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-0-xs"
              name="dress-us-petite-0-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-0-xs"
            >
              0, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-2-xs"
              name="dress-us-petite-2-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-2-xs"
            >
              2, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-4-s"
              name="dress-us-petite-4-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-4-s"
            >
              4, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-6-s"
              name="dress-us-petite-6-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-6-s"
            >
              6, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-8-m"
              name="dress-us-petite-8-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-8-m"
            >
              8, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-10-m"
              name="dress-us-petite-10-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-10-m"
            >
              10, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-12-l"
              name="dress-us-petite-12-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-12-l"
            >
              12, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-14-l"
              name="dress-us-petite-14-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-14-l"
            >
              14, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-16-XL"
              name="dress-us-petite-16-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-16-XL"
            >
              16, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-18-XL"
              name="dress-us-petite-18-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-18-XL"
            >
              18, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-20-2X"
              name="dress-us-petite-20-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-20-2X"
            >
              20, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-22-2X"
              name="dress-us-petite-22-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-22-2X"
            >
              22, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-24-3X"
              name="dress-us-petite-24-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-24-3X"
            >
              24, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-26-3X"
              name="dress-us-petite-26-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-26-3X"
            >
              26, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-petite-28-4X"
              name="dress-us-petite-28-4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-petite-28-4X"
            >
              28, 4X
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Tall US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-00-XXS"
              name="dress-us-tall-00-XXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-00-XXS"
            >
              00, XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-0-xs"
              name="dress-us-tall-0-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-0-xs"
            >
              0, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-2-xs"
              name="dress-us-tall-2-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-2-xs"
            >
              2, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-4-s"
              name="dress-us-tall-4-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-4-s"
            >
              4, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-6-s"
              name="dress-us-tall-6-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-6-s"
            >
              6, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-8-m"
              name="dress-us-tall-8-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-8-m"
            >
              8, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-10-m"
              name="dress-us-tall-10-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-10-m"
            >
              10, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-12-l"
              name="dress-us-tall-12-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-12-l"
            >
              12, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-14-l"
              name="dress-us-tall-14-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-14-l"
            >
              14, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-16-XL"
              name="dress-us-tall-16-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-16-XL"
            >
              16, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-18-XL"
              name="dress-us-tall-18-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-18-XL"
            >
              18, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-20-2X"
              name="dress-us-tall-20-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-20-2X"
            >
              20, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-22-2X"
              name="dress-us-tall-22-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-22-2X"
            >
              22, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-24-3X"
              name="dress-us-tall-24-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-24-3X"
            >
              24, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-26-3X"
              name="dress-us-tall-26-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-26-3X"
            >
              26, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-tall-28-4X"
              name="dress-us-tall-28-4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-tall-28-4X"
            >
              28, 4X
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Maternity US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-00-XXS"
              name="dress-us-maternity-00-XXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-00-XXS"
            >
              00, XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-0-xs"
              name="dress-us-maternity-0-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-0-xs"
            >
              0, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-2-xs"
              name="dress-us-maternity-2-xs"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-2-xs"
            >
              2, XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-4-s"
              name="dress-us-maternity-4-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-4-s"
            >
              4, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-6-s"
              name="dress-us-maternity-6-s"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-6-s"
            >
              6, S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-8-m"
              name="dress-us-maternity-8-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-8-m"
            >
              8, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-10-m"
              name="dress-us-maternity-10-m"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-10-m"
            >
              10, M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-12-l"
              name="dress-us-maternity-12-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-12-l"
            >
              12, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-14-l"
              name="dress-us-maternity-14-l"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-14-l"
            >
              14, L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-16-XL"
              name="dress-us-maternity-16-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-16-XL"
            >
              16, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-18-XL"
              name="dress-us-maternity-18-XL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-18-XL"
            >
              18, XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-20-2X"
              name="dress-us-maternity-20-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-20-2X"
            >
              20, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-22-2X"
              name="dress-us-maternity-22-2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-22-2X"
            >
              22, 2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-24-3X"
              name="dress-us-maternity-24-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-24-3X"
            >
              24, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-26-3X"
              name="dress-us-maternity-26-3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-26-3X"
            >
              26, 3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id="dress-us-maternity-28-4X"
              name="dress-us-maternity-28-4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor="dress-us-maternity-28-4X"
            >
              28, 4X
            </label>
          </div>
        </div>
      </div>
    </form>
  </section>
  <footer
    className="NPFFooter"
  >
    <button
      className="NPFFooter_previous"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-left fa-w-14 NPFFooter_arrow"
        data-icon="arrow-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span>
        PREVIOUS
      </span>
    </button>
    <button
      className="NPFFooter_next"
      onClick={[Function]}
    >
      <span>
        NEXT
      </span>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-right fa-w-14 NPFFooter_arrow"
        data-icon="arrow-right"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </footer>
</div>
`;
